main.py

```diff
@@ -0,0 +1,17 @@
+import tensorflow as tf
+from lstm import Config
+from lstm import Lstm
+
+def test_LstmModel():
+  config = Config()
+  config.data_path = "data/sentences_debug.train"
```

Comment:
Why is this hard coded? I thought we were using commandline parameters for this stuff. Also, this file is not standard...